docs/static/ecs-compatibility.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[ecs-compatibility]]
+=== ECS compatibility
+
+:ecs-major-version: v1
+
+Many plugins implement an ECS-compatibility mode, which causes them to produce and manipulate events in a manner that is compatible with the Elastic Common Schema (ECS).
+
+Any plugin that supports this mode will also have an `ecs_compatibility` option, which allows you to configure which mode the individual plugin instance should operate in.
+If left unspecified for an individual plugin, the pipeline's `pipeline.ecs_compatibility` setting will be observed.
+This allows you to configure plugins to use their legacy non-ECS behaviour: individually, per-pipeline, or globally.
+
+ECS Compatibility modes do not prevent you from explicitly configuring a plugin in a manner that conflicts with ECS.
+Instead, they ensure that _implicit_ configuration avoids conflicts.
+
+NOTE: This pre-release branch of {ls} 8 defaults to ECS v{ecs-major-version}, but by the time {ls} 8 ships, it will point to ECS v8.
+We expect the scope of breaking changes in ECS 8 to be limited, but progress toward the definition of ECS v8 can be tracked https://github.com/elastic/ecs/issues/839[here].
+
+NOTE: Until {ls} 8.0 and the final 7.x are released, any value for `pipeline.ecs_compatibility` other than `disabled` -- _including the default value on pre-release builds_ -- are considered BETA and unsupported.
+As we continue to release plugins with ECS Compatibility modes, having this flag set will cause upgrades to _automatically_ consume breaking changes from one snapshot to another, changing the shape of data the plugin produces.
+If you require stability while testing against the unreleased {ls} 8, we encourage you to opt-out globally or per-pipeline as detailed below.
+
+==== Opting out of ECS
+
+
+In {ls} 8, these plugins are run in ECS mode by default, but you can opt-out and use the legacy behaviour in one of several ways.
+This can be helpful if you have very complex pipelines that were defined pre-ECS, to allow you to either upgrade them or to avoid doing so independently of your {ls} 8.x upgrade.
+
+===== Specific plugin instance
+
+Use a plugin's `ecs_compatibility` option to override the default value on the plugin instance.
+
+For example, if you want a specific instance of the GeoIP Filter to behave without ECS compatibility, you can adjust its definition in your pipeline without affecting any other plugin instances.
+
+[source,text]
+filter {
+  geoip {
+    source => "[host][ip]"
+    ecs_compatibility => disabled
+  }
+}
```

Comment:
```suggestion
[source,text]
-----
filter {
  geoip {
    source => "[host][ip]"
    ecs_compatibility => disabled
  }
}
-----
```

Reply:
The code sample works without the delimiters, but this is a standard we try to follow.